tests/test_timereport.py

```diff
@@ -1,8 +1,8 @@
 import os
 import datetime
-from timereport.lib.helpers import parse_config
-from timereport.lib.slack import slack_payload_extractor
-from timereport.lib.factory import factory
+from timereport.chalicelib.lib.helpers import parse_config
```

Comment:
Why is `chalicelib` a better name than `lib`?

Reply:
@kamigerami All right. Then maybe we should remove `lib` directory and put the files in `chalicelib` instead. It feels a bit much to have `chalicelib/lib`.
Just my opinion though. Do as you like 😄